apprise_test.go

```
@@ -0,0 +1,119 @@
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestAppriseSendJSONTarget(t *testing.T) {
+	requests := make(chan map[string]any, 1)
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer r.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `r.Body.Close` is not checked

(errcheck)